trace.py

```diff
@@ -195,7 +195,31 @@ def define_val_type(val): # recursive function used to change strings into typed
 # Load binary passed in argv[1] and check we only have one arg
 if len(sys.argv) < 2:
     print("Please pass in at least one argument (executable you wish to run)!")
+    os.remove("output.txt")
     exit()
+# Test whether this file exists by checking if the path exists
+if not os.path.exists(sys.argv[1]):
+    print("This executable does not exist!")
+    os.remove("output.txt")
```

Comment:
I think all the error checking statement here should belong in one function

Reply:
that way we have one function call to like `check_executable_good` or something and it makes it all more clear.